tasks_downstream.py

```diff
@@ -244,6 +244,24 @@ def write_code_workspace_file(c, cw_path=None):
                 "problemMatcher": [],
                 "options": {"statusbar": {"label": "$(play-circle) Start Odoo"}},
             },
+            {
+                "label": "Install current module",
+                "type": "process",
+                "command": "invoke",
+                "args": ["install", "--cur-file", "${file}"],
```

Comment:
How about this?

```suggestion
                "args": ["install", "--cur-file", "${file}", "restart"],
```

So, Odoo is up and running after install?